projects/components/src/multi-select/multi-select.component.ts

```diff
@@ -92,7 +92,11 @@ import { MultiSelectJustify } from './multi-select-justify';
                 (click)="this.onSelectionChange(item)"
                 class="multi-select-option"
               >
-                <ht-checkbox class="checkbox" [checked]="this.isSelectedItem(item)"></ht-checkbox>
+                <ht-checkbox
```

Comment:
We actually already have a component that handles this for us - look for `<ht-event-blocker event="click">`

Reply:
But now that I look at this isn't the issue that we're listening on the parent to broadcast the change? So wouldn't blocking this event prevent the broadcast?

Reply:
In this case, I'm calling to prevent-default function to block the click on the checkbox allowing to call the parent function because the values are set from the parent component.

Reply:
we need to use checkedChange right?

Reply:
And possibly put `<ht-event-blocker event="click">` inside checkbox?

Reply:
Clarified offline - we're preventing default but not stopping propagation. Given that, the event blocker is probably more of a pain to use (we'd have to wire up checkbox's change listener in addition to the outer click listener), so I'm ok sticking with this approach - but let's clarify the naming a bit: `(click)="this.preventClickDefault($event)"` or similar (since propagation isn't being blocked)

Reply:
@aaron-steinfeld would one of the following also work? 
```
- Put ht-event-blocker inside ht-checkbox component. then click won’t be propagated. Now use `checkedChange` output in multi select to call appropriate selection method
- Inside multi select, wrap  `<ht-event-blocker><ht-checkbox></ht-checkbox></ht-event-blocker> `. Now use `checkedChange`  output in multi select to call appropriate selection method
```

Reply:
I tested both approaches and they are blocking the propagation of the checkbox, so the  checkbox value is not updated

Reply:
Either should work - but requires using both a click listener (to get label clicks) and the checkedChange to get checkbox clicks. This is what I was getting at above - it's doable, but just using prevent default ends up cleaner in this situation. The real ideal fix is a bigger overhaul of checkbox to support more complex labels so it handles the whole click internally, but out of scope IMO.

Reply:
done